### PR TITLE
fix(config): set user agent and service name, add version package

### PR DESCRIPTION
**Pull request by sanderblue**
Resolves: #89

### CODE REVIEW

File: build/test.mk [hunk 1]
```diff
@@ -25,7 +25,7 @@ test: test-only
 test-only: test-unit test-integration
 
 test-unit:
-	@echo "=== $(PROJECT_NAME) === [ test-unit        ]: running unit tests..."
+	@echo "=== $(PROJECT_NAME) ${PROJECT_VER_TAGGED} === [ test-unit        ]: running unit tests..."
```
Comment (jthurman42) [inline build/test.mk]: Why add the version tag (and break the beautiful output format followed by the rest of the make system?

Comment (sanderblue) [reply build/test.mk]: I wasnt done. I should've opened a draft PR lol

File: cmd/newrelic/main.go [hunk 2]
```diff
@@ -8,13 +8,14 @@ import (
...
 	"github.com/newrelic/newrelic-cli/internal/entities"
+	"github.com/newrelic/newrelic-cli/internal/version"
 )
 
 var (
 	// AppName for this CMD
 	AppName = "newrelic-dev"
 	// Version of the CLI
-	Version = "dev"
+	Version = version.Version
```
Comment (jthurman42) [inline cmd/newrelic/main.go]: We don't need to set another global here, just pass it to `Execute` on line 39, or better yet is to remove completely from `main.go` and put directly into `command.go` inside of `Command`, and fix `versionCmd` to reference as well.

File: internal/client/client.go [hunk 3]
```diff
@@ -15,6 +16,7 @@ func CreateNRClient(cfg *config.Config, creds *credentials.Credentials) (*newrel
 		err            error
 		personalAPIKey string
 		region         string
+		serviceName    = "newrelic-cli"
```
Comment (jthurman42) [inline internal/client/client.go]: This should be a constant in the package, easier to find / change later.

File: internal/version/version.go [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package version
+
+// Version of the CLI
+const Version string = "0.1.0"
```
Comment (jthurman42) [inline internal/version/version.go]: Add the Release makefile / scripts to automatically update this ala https://github.com/newrelic/newrelic-client-go/blob/master/build/release.mk

Comment (sanderblue) [reply internal/version/version.go]: This felt out of scope for this PR, but I went ahead and did it.

**Pull request merged**